clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/InfinispanLogger.java

```diff
@@ -111,6 +111,10 @@ public interface InfinispanLogger extends BasicLogger {
 //    @Message(id = 8, value = "Could not resolve destination address for outbound socket binding named '%s'")
 //    InjectionException failedToInjectSocketBinding(@Cause UnknownHostException cause, OutboundSocketBinding binding);
 
+    @LogMessage(level = WARN)
+    @Message(id = 9, value = "The '%s' element is no longer supported and will be ignored")
+    void elementIgnored(String element);
```

Comment:
We cannot reuse message IDs that have been already used for different messages (that's why we comment them out to prevent this - looks like this was missed). The ID 9  used to be

```
  @Message(id = 9, value = "Failed to add %s %s cache to non-clustered %s cache container.")
    StartException transportRequired(CacheMode mode, String cache, String cacheContainer);
```

https://github.com/wildfly/wildfly/blob/9.x/clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/InfinispanLogger.java#L123-L124

Move it as message ID 34.

Reply:
ID 9 has not been used since WF9.  I think it is safe to reuse at this point.

Reply:
Let's hope, but until we run out of our allotted range, I don't see the reason to prematurely reuse.

Reply:
I don't need to hope. :) It is safe to reuse.

Reply:
I believe you just proved my point! :laughing: